biosimulations/apps/dispatch/src/app/services/simulation/simulation.service.ts

```diff
@@ -93,32 +93,38 @@ export class SimulationService {
       .map((simulation: Simulation): string => {
         return simulation.id;
       })
-      .join(',');
+      
+
+      const promises = [];
+      for(const simId of simulationIds) {
+        const promise = this.httpClient.get(`${urls.dispatchApi}run/${simId}`).toPromise();
+        promises.push(promise);
+      }
+
+      Promise.all(promises).then((data: any) => {
 
-    this.httpClient.post(urls.dispatchApi + 'jobinfo', simulationIds.split(',')).subscribe(
-      (data: any) => {
-        // Reformatting simulations according to Simulation interface
         const simulations: Simulation[] = [];
-        for (const sim of data.data) {
-          const dispatchSim: DispatchSimulationModelDB = sim;
+        for (const sim of data) {
+          const dispatchSim = sim;
           simulations.push({
             name: dispatchSim.name,
             email: dispatchSim.email,
-            runtime: dispatchSim.runtime,
-            id: dispatchSim.uuid,
+            //runtime: dispatchSim.runtime,
```

Comment:
Why is `runtime` and `resultsSize` commented out? I have the same question about L180/189. If these are commented out, the runtime and result size won't get updated.

Reply:
Now runTime and ResultsSize are working